docs/design/features/OsrDetailsAndDebugging.md

```diff
@@ -172,7 +172,7 @@ During the patchpoint phase, if any block was marked, the jit adds a
 new integer local to the method (the patchpoint counter) and adds IR
 to initialize the counter on method entry to the value of
 `DOTNET_TC_OnStackReplacement_InitialCounter` (by default this is
-0x1000).
+1000).
```

Comment:
0x1000 should be more correct, all config switches accept inputs in hex format, so e.g. in jitconfigvalues.h
```
CONFIG_INTEGER(TC_OnStackReplacement_InitialCounter, W("TC_OnStackReplacement_InitialCounter"), 1000)
```
means the counter is actually 4096

Reply:
The default here is treated as an int literal, so `1000` as the default will be a thousand

https://github.com/dotnet/runtime/blob/8c4e6e2aba31a716e60f8546010b09d848f202d2/src/coreclr/jit/jitconfig.cpp#L434-L441

Reply:
Ah, indeed, just realized that 🙂